src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx

```diff
@@ -89,18 +91,29 @@ export function VideoEmbedInnerNative({
 }
 
 function VideoControls({
-  player,
   enterFullscreen,
   timeRemaining,
   isMuted,
 }: {
-  player: VideoPlayer
   enterFullscreen: () => void
   timeRemaining: number
   isMuted: boolean
 }) {
   const {_} = useLingui()
   const t = useTheme()
+  const {player, setMutedInFeed} = useActiveVideoNative()
+  const autoplayDisabled = useAutoplayDisabled()
+  const [isPlaying, setIsPlaying] = React.useState(player.playing)
+
+  useEffect(() => {
+    function listener(newIsPlaying: boolean) {
+      setIsPlaying(newIsPlaying)
+    }
+    player.addListener('playingChange', listener)
+    return () => {
+      player.removeListener('playingChange', listener)
+    }
+  }, [player])
```

Comment:
We've had issues with player in useeffects before, is this one ok?

Reply:
![Screenshot 2024-09-12 at 3 30 40 PM](https://github.com/user-attachments/assets/374bf4dd-bcaa-412c-8dad-9e66f527990e)

🥳 (can update this after merging my fix or vice versa, but this effect should be okay in general from what i saw before...)